website/content/partials/known-issues/aws-auth-external-id.mdx

```diff
@@ -0,0 +1,14 @@
+### AWS Auth Role configuration requires an external_id
```

Comment:
```suggestion
### AWS authN role configuration requires an external ID
```

Reply:
Does this suggestion mean to include the capital N?

Reply:
Yes. "auth" can be ambiguous, so we include the 'N' or 'Z' to differentiate between authentication (authN) and authorization (authZ)